handbook/engineering/README.md

```diff
@@ -440,6 +440,40 @@ The on-call developer is responsible for:
 3. Comment on the GitHub issue and at-mention the PM and anyone listed in the requester field.
 4. If `customer-` labels are applied to the user story, at-mention the [VP of Customer Success](https://fleetdm.com/handbook/customer-success#team).
 
+### Make changes to the vulnerabilities and nvd repositories
+The [nvd](https://github.com/fleetdm/nvd) and [vulnerabilities](https://github.com/fleetdm/vulnerabilities) repositories continuously create output files that are downloaded and used by our customers' Fleet servers. 
+They run code that is managed in two places: 
+1. The code and workflows in their respective repository.
```

Comment:
```suggestion
- The code and workflows in their respective repository.
```
Changing bullets to an unordered list since we're not listing sequential steps